src/swift/DispatchStubs.cc

```diff
@@ -173,12 +173,6 @@ _swift_dispatch_retain(dispatch_object_t obj) {
   dispatch_retain(obj);
 }
 
-// DISPATCH_RUNTIME_STDLIB_INTERFACE
-// extern "C" dispatch_queue_t
-// _swift_apply_current_root_queue() {
-// 	return DISPATCH_APPLY_CURRENT_ROOT_QUEUE;
-// }
-
```

Comment:
Should this also be taken out of Queue.swift in the overlay?  There's a silgen directive for this symbol towards the bottom of that file.